VnE/Source/Extensions/ChemPack/parsers.py

```
@@ -52,32 +51,19 @@ def fracToDec(a, b, c, al, be, ga, coords):
                         [0, b*sin(ga), c*(cos(al)-cos(be)*cos(ga))/sin(ga)],
                         [0, 0, c*p/sin(ga)]])
         for i in range(len(coords)):
             coord = np.array(coords[i])[...,np.newaxis]
             coord = mat @ coord
             coord = coord.transpose().squeeze()
-            #coords[i] = (coords[i] @ mat).astype(dtype=np.float32)
             coords[i] = coord
         return coords
 
-    def parsFile(self, file_path, bond=True, root=None):
+    def parsFile(self, file_path, bond=True, root=None) -> tuple[MoleculeClass.MoleculeSystem, tuple[point_class.PointsList]]:
```

Ruff (0.15.2)

[error] Boolean default positional argument in function definition

(FBT002)